tests/Resources/ti.ui.window.test.js

```diff
@@ -18,12 +18,7 @@ describe('Titanium.UI.Window', function () {
 	let win;
 	afterEach(done => { // fires after every test in sub-suites too...
 		if (win && !win.closed) {
-			win.addEventListener('close', function listener () {
-				win.removeEventListener('close', listener);
-				win = null;
-				done();
-			});
-			win.close();
+			win.close().then(() => done()).catch(_e => done());
```

Comment:
<!--
  0 failure: 
  2 warning:  tests/Resources/t..., tests/Resources/t...
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.window.test.js;
  Line: 21;
-->

- :warning: tests/Resources/ti.ui.window.test.js line 21 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)
- :warning: tests/Resources/ti.ui.window.test.js line 21 – Avoid calling back inside of a promise. (promise/no-callback-in-promise)